build-tools/feeds/internal-build.txt

```diff
@@ -0,0 +1,93 @@
+@fluidframework/tool-utils
```

Comment:
It would be good (separate PR?) to also add checks to make sure what we publish a complete set (i.e. check the dependencies and make sure they are publish to the same feed or feed that are intended to be combined with (for dev and test feeds)

Reply:
Good point. I'll consider a separate PR.